app/presenters/tree_node/middleware_server.rb

```diff
@@ -0,0 +1,5 @@
+module TreeNode
+  class MiddlewareServer < Node
+    set_attribute(:title) { [@object.name.presence, @object.host_name].join(" - ") }
```

Comment:
Looks like `middleware_server` never had a `host_name`, where does this come from?